filter/ArticleMedraXmlFilter.php

```diff
@@ -304,8 +324,9 @@ function createContentItemNode(DOMDocument $doc, Issue $issue, Submission $artic
         $allKeywords = $article->getCurrentPublication()->getData('keywords');
         $keywords = $this->getPrimaryTranslation($allKeywords, $objectLocalePrecedence);
         if (!empty($keywords)) {
-            $keywordsString = implode(';', $keywords);
-            $contentItemNode->appendChild($this->createSubjectNode($doc, self::O4DOI_SUBJECT_SCHEME_PUBLISHER, $keywordsString));
+            foreach ($keywords as $keyword) {
+                $contentItemNode->appendChild($this->createSubjectNode($doc, self::O4DOI_SUBJECT_SCHEME_PUBLISHER, $keyword));
```

Comment:
It should be O4DOI_SUBJECT_SCHEME_KEYWORDS, right?

Reply:
Hi @bozana , I fixed this issue. 
Thanks,
@njikonga-med